examples/dmd/heat_conduction_dmdc.cpp

```diff
@@ -0,0 +1,717 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2023, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+//      libROM MFEM Example: DMDc for Heat_Conduction (adapted from ex16p.cpp)
+//
+// Compile with: make heat_conduction_dmdc
```

Comment:
There is a lot of duplication between this file and `heat_conduction.cpp`. Could they be combined, or would that make things too complicated?

Reply:
I recommend the two files be separated as the two examples have some fundamental difference. Combining them may easily make the existing DMD example `heat_conduction` breaks, and more importantly, make further development on `heat_conduction_dmdc` difficult. They use different problem settings, and different classes, `DMD` and `DMDc` which is hard to unify into a parent class. We had the `DMD` class from which `AdaptiveDMD` and `NonuniformDMD` derives, and the `ParametricDMD` on top of that which interpolates matrices in `DMD`. Right now, I have not much idea how adpative/nonuniform version of DMDc should be implemented, but parametric version of DMDc will have to interpolate different matrices than DMD.